src/apim_checkout.tf

```diff
@@ -3,7 +3,7 @@
 ##############
 
 module "apim_checkout_product" {
-  count  = var.checkout_enabled ? 1 : 0
+  count  = var.checkout_enabled || var.pagopa_proxy_enabled ? 1 : 0
```

Comment:
why `||` ? 
it shouldn't be `&&` ?

Reply:
`module.checkout_function` is enabled through `checkout_enabled`, while `pagopa-proxy` infra is enabled through `pagopa_proxy_enabled` and both need on `apim_checkout_product`.

Should the proxy or the checkout function not be enabled this product must be present in order to expose everything to the APIM correctly (at least for how things are configured now).